zeppelin-web-angular/src/index.html

```diff
@@ -44,6 +43,14 @@ <h1 class="brand-title" style="display: none">Zeppelin</h1>
         "HTML-CSS": { availableFonts: ["TeX"] },
         messageStyle: "none"
       }
+      const origin = window.location.origin;
```

Comment:
Why is this needed?

Reply:
MathJax resolves its dependency URLs based on its configuration.
Previously, it was loading its dependency files from the root URL (the default UI app URL), regardless of whether it was being served from the default UI or the `/next` UI.
Since both apps included MathJax as a dependency, this didn't cause any issues, but I felt it was more appropriate for MathJax to load its dependencies from the correct app-specific paths.

So, I updated th MathJax configurations to ensure it resolves URLs relative to the correct app.